app/app.test.js

```diff
@@ -293,4 +291,27 @@ describe('GET',() => {
                 })   
            })
         });
-      })
\ No newline at end of file
+      })
+
+      describe('GET',() => {
+        describe('/api/articles', () => {
```

Comment:
Don't you already have a /api/articles describe block ? Why does this test need another one?